pkg/asset/cluster/tfvars.go

```diff
@@ -378,6 +378,17 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+
+		instanceServiceAccount := ""
+		if installConfig.Config.CredentialsMode == types.PassthroughCredentialsMode {
+			serviceAccount := make(map[string]interface{})
+			err := json.Unmarshal([]byte(sess.Credentials.JSON), &serviceAccount)
+			if err != nil {
+				return err
+			}
+			instanceServiceAccount = serviceAccount["client_email"].(string)
```

Comment:
It might be beneficial to add some error handling here for when "client_email" is not found.
```
instanceServiceAccount, found = serviceAccount["client_email"].(string)
if !found {
    return errors.new("...")
}
```